Tests/APIKit/BodyParametersType/MultipartFormDataParametersTests.swift

```diff
@@ -1,6 +1,6 @@
 import Foundation
 import XCTest
-@testable import APIKit
```

Comment:
Sounds cool. I've added the build setting to CI job and restored access control modifier of the Data initializer in 66ace22b427f432d9e62d8a52f4afb01cc8c9330.